rest/data/perf.go

```diff
@@ -265,6 +265,30 @@ func (dbc *DBConnector) ScheduleSignalProcessingRecalculateJobs(ctx context.Cont
 	return catcher.Resolve()
 }
 
+type ChangePointStub struct {
+	PerfResultID string `bson:"perf_result_id" json:"perf_result_id" yaml:"perf_result_id"`
+	Measurement  string `bson:"measurement" json:"measurement" yaml:"measurement"`
+}
+
+func (dbc *DBConnector) TriageChangePoints(ctx context.Context, changePoints map[string]string, status string) error {
+	ts := model.TriageStatus(status)
+	err := ts.Validate()
+	if err != nil {
+		return gimlet.ErrorResponse{
+			StatusCode: http.StatusUnprocessableEntity,
```

Comment:
is this something that users should retry?

Reply:
Not automatically; this is basically invalid input and they need to fix their request.